### DISCUSSION TITLE
Solo pool testing

### DISCUSSION THREAD

**Discussion by onemorebsmith** [onemorebsmith/kaspa-stratum-bridge#34] [General]
Thread for reporting issues with the solo pool

Please include miner logs if possible!

**Comment by Chadwicksracing** [comment]
![image](https://user-images.githubusercontent.com/42011043/192413111-ec0e5bd0-48ea-49ee-9373-c0ad04c94eb5.png)
Getting some shares, but the rest are bad

**Reply by onemorebsmith** [reply]
Gotcha, I'll take a look. What miner is that? And what version?

**Reply by Chadwicksracing** [reply]
SRB Latest v

**Reply by onemorebsmith** [reply]
10-4. I'll give it a look. Thanks dude

**Comment by Chadwicksracing** [comment]
![image](https://user-images.githubusercontent.com/42011043/192416055-7d331a41-f473-4974-8f41-5217981b931b.png)
LOLminer Latest v

**Reply by onemorebsmith** [reply]
Removed stale tracking, it should be good to go now.

**Comment by Chadwicksracing** [comment]
Good to go on errors in miner.

**Reply by onemorebsmith** [reply]
😎 nice, thanks for letting me know :)

**Comment by rpringst** [comment]
Is there a means to view block hashes of mined blocks? I wish to locate a block I mined on the explorer.

**Reply by onemorebsmith** [reply]
you can look at your mining address in the new explorer https://explorer.kaspa.org/ and all the blocks should be there

**Reply by rpringst** [reply]
It absolutely is not there. I don't understand.

**Reply by rpringst** [reply]
@onemorebsmith Do you have the block hashes for blocks mined on Sept 29th, 2022? It would have been a little bit before 4PM Central Time.

**Reply by onemorebsmith** [reply]
What is your wallet address? I have logs from a block submission at 16:52:13.218 EDT for block `2f509e6c60a42cb3259757519943d968ee8d626c9792f7c18a7cb16137e55e7a`

https://katnip.cbytensky.org/block/2f509e6c60a42cb3259757519943d968ee8d626c9792f7c18a7cb16137e55e7a

**Reply by onemorebsmith** [reply]
For posterity's sake, the issue here was that he mined a 'red' block which doesn't have a mining reward associated. The block in question: https://kgi.kaspad.net/?hash=2f509e6c60a42cb3259757519943d968ee8d626c9792f7c18a7cb16137e55e7a

**Comment by agarwood777** [comment]
Hello, I would like to ask if I see the BLOCK on the dashboard, it has been 5 hours, but the wallet still does not see the entry
![BLOCK 26976263 2022-10-05 232409](https://user-images.githubusercontent.com/83913710/194113007-3fe425ab-6e9f-4a92-9955-c651c8d1acf8.jpg)

**Reply by onemorebsmith** [reply]
You might have mined a red block, see the discussion above.

**Reply by onemorebsmith** [reply]
Actually no, here's your block: https://explorer.kaspa.org/blocks/048c65d69ab86840defc3eb2f3a8d349db1c4fb79a176b4494eca94fa5f05489

The 'mined by' isn't your wallet, so not sure what happened there.

**Reply by onemorebsmith** [reply]
Do you happen to have a log file or anything?

**Comment by agarwood777** [comment]
ok i understand thank you

Brandon Smith ***@***.***> 於 2022年10月6日 週四 凌晨12:54寫道：

**Comment by agarwood777** [comment]
In addition, I would like to ask why the address of the KAS wallet is
automatically changed?

steven huang ***@***.***> 於 2022年10月6日 週四 凌晨12:55寫道：
